code/modules/mob/living/carbon/human/species_types/synthliz.dm

```diff
@@ -0,0 +1,51 @@
+/datum/species/synthliz
```

Comment:
Main reason they aren't an IPC subtype is due to IPCs being hardcoded to have screens. That, and their "robot that's intentionally limited to be worse than or on par with organics to prevent potential uprisings" status in most synth lore interpretations would make it awkward if they inherited all potential mechanics for IPCs.

Reply:
Ouch, are the IPC screens really that much hardcoded?